src/test/java/business/CommonSteps.java

```diff
@@ -0,0 +1,18 @@
+package business;
+
+import core.Config;
+import org.testng.annotations.Test;
+
+import static core.Config.logger;
+
+public class CommonSteps {
+    @Test
+    public void iInvokeLoginPageByUrl() {
+        logger.info("test starts");
+        logger.info("Init config file");
+        Config.initFile();
```

Comment:
create separate class for storing properties as a constants. Please avoid direct reading properties in your code

Reply:
fixed